README.md

```diff
@@ -14,13 +14,18 @@ To install, follow the detailed installation instructions at http://wiki.volksza
 
 If you're impatient you can quickstart using (Debian Bullseye or Ubuntu 18.04 LTS):
 
-    sudo apt-get install build-essential git-core cmake pkg-config subversion libcurl3-dev \
-      libgnutls-dev libsasl2-dev uuid-dev uuid-runtime libtool dh-autoreconf libunistring-dev
+    sudo apt-get install build-essential git-core cmake pkg-config subversion libcurl4-dev \
+      gnutls-dev libsasl2-dev uuid-dev uuid-runtime libtool dh-autoreconf libunistring-dev \
+      libjson-c-dev
```

Comment:
we have some code that uses openssl directly, you can see this here for example: https://github.com/volkszaehler/vzlogger/pull/530/files

and then there's the ssl library used by curl.
we might benefit from using an openssl-backed libcurl to reduce the memory footprint.

i wonder why you are seeing `NOTFOUND` there, because there's `libcurl4-dev` on the list, which should pull in SOME version of libcurl, (and when adding libcurl-*ssl it becomes redundant.)    
i was hoping to avoid demanding a specific ssl library for curl here.
(was also wondering how users would deal with maybe ending up with a not-ssl-enabled version of libcurl - if any distro ships such a thing. i think most usage is via http to localhost anyway.)

(i guess you're not seriously proposing to add a second copy of libunistring-dev, and that's just a copy/paste error.)

Reply:
To recap:
I have been compiling vzlogger by having a look on the instructions on the master branch. And during that I installed missing things and noted down what I installed to get things running. Then I recognized that there is already a PR considering updating the dependencies. I tried to merge my changes into the list. Could happen that I duplicated / missed something by manually merging my findings into it. So does not have to be perfect what I noted as a comment.